File: Laboratorio de desenvolvimento III - /backend/java/sbur-rest-demo/src/main/java/com/thehecklers/sburrestdemo/model/EmpresaModel.java
package com.thehecklers.sburrestdemo.model;

public class EmpresaModel {

    private Long id;
    private String nome;
    private String ramo;

    public EmpresaModel() {
        // Construtor padrão necessário para deserialização JSON
    }

    public EmpresaModel(Long id, String nome, String ramo) {
        this.id = id;
        this.nome = nome;
        this.ramo = ramo;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getRamo() {
        return ramo;
    }

    public void setRamo(String ramo) {
        this.ramo = ramo;
    }
}

File: Laboratorio de desenvolvimento III - /frontend/src/model/EmpresaModel.ts
// EmpresaModel.ts
export class EmpresaModel {
    id: number;
    nome: string;
    ramo: string;

    constructor(nome: string, ramo: string, id: number = 0) {
        this.id = id;
        this.nome = nome;
        this.ramo = ramo;
    }
}
